pallets/frequency-tx-payment/src/tests/stable_weights_tests.rs

```diff
@@ -10,51 +10,51 @@ fn test_weights_are_stable() {
 			(
 				"create_sponsored_account_with_delegation",
 				SubstrateWeight::<Test>::create_sponsored_account_with_delegation(100),
-				136951617,
+				1130383617,
```

Comment:
These value were no correct before because it was using `0` for DB **read** and DB **write**s.